src/main/app/src/app/admin/parameter-edit/smart-documents/smart-documents-tree.component.html

```diff
@@ -24,6 +29,12 @@
         }
       </mat-select>
     </mat-form-field>
+    <div
+      [ngClass]="{'visible': zaaktypeVertrouwelijkheid(node.informatieObjectTypeUUID)}"
```

Comment:
Just wondering: is it good practise to add this kind of 'retrieval logic' in these .html component templates? Is it not better to do this in the component's TypeScript file and only read out a variable here?

Reply:
Good point; now solved with div wrapper that takes fixed with and its content depending on availability and *ngIf'd. Better right?

Reply:
However, the build is failing.. ;-)